[BUG] from_pytest_fixture doesn't seem to work with parametrized fixtures

When using ```schemathesis.from_pytest_fixture()```, if that fixture is parametrized with ```@pytest.fixture(params=...)```, pytest produces an error like this:
```
tests/integration/test_graphql_schema.py::test_graphql_schema FAILED                                                                                                                           [100%]

============================================================================================== FAILURES ==============================================================================================
________________________________________________________________________________________ test_graphql_schema _________________________________________________________________________________________
'Function' object has no attribute 'callspec'

During handling of the above exception, another exception occurred:
The requested fixture has no parameter defined for test:
    tests/integration/test_graphql_schema.py::test_graphql_schema

Requested fixture 'graphql_schema_fixture' defined in:
tests/integration/test_graphql_schema.py:8

Requested here:
./venv/lib/python3.8/site-packages/schemathesis/lazy.py:189
```

**To Reproduce**
Here's an example of what the type of tests I'm trying to run:
```
import pytest
import schemathesis

API_VERSIONS = ["v1", "v2", "v3"]

@pytest.fixture(params=API_VERSIONS)
def graphql_schema_fixture(app):
    endpoint = f"/{request.param}/graphql"
    return schemathesis.graphql.from_wsgi(endpoint, app)

graphql_schema = schemathesis.from_pytest_fixture("graphql_schema_fixture")

@graphql_schema.parametrize()
def test_graphql_schema(case):
    response = case.call_wsgi()
    case.validate_response(response)
```

**Expected behavior**
When generating a schema object from a parametrized fixture, decorating a test function with that schema object would result in multiple iterations of the test function, for each schema produced by the parametrized fixture.

**Environment (please complete the following information):**
 - OS: Linux
 - Python version: 3.8.5
 - Schemathesis version: 3.6.0
 - Spec version: Open API 3.0.2
 - Pytest version: 6.2.3

--- a/docs/changelog.rst
+++ b/docs/changelog.rst
@@ -4,6 +4,10 @@ Changelog
 `Unreleased`_ - TBD
 -------------------
 
+**Fixed**
+
+- Using parametrized ``pytest`` fixtures with the ``from_pytest_fixture`` loader. `#1121`_
+
 `3.6.0`_ - 2021-04-04
 ---------------------
 
@@ -1811,6 +1815,7 @@ Deprecated
 .. _0.3.0: https://github.com/schemathesis/schemathesis/compare/v0.2.0...v0.3.0
 .. _0.2.0: https://github.com/schemathesis/schemathesis/compare/v0.1.0...v0.2.0
 
+.. _#1121: https://github.com/schemathesis/schemathesis/issues/1121
 .. _#1100: https://github.com/schemathesis/schemathesis/issues/1100
 .. _#1097: https://github.com/schemathesis/schemathesis/issues/1097
 .. _#1093: https://github.com/schemathesis/schemathesis/issues/1093

--- a/src/schemathesis/lazy.py
+++ b/src/schemathesis/lazy.py
@@ -2,6 +2,7 @@ from inspect import signature
 from typing import Any, Callable, Dict, Iterable, Optional, Union
 
 import attr
+import pytest
 from _pytest.fixtures import FixtureRequest
 from pytest_subtests import SubTests, nullcontext
 
@@ -120,6 +121,8 @@ class LazySchema:
                     else:
                         _schema_error(subtests, result.err(), node_id, data_generation_method)
                 subtests.item._nodeid = node_id
+
+            test = pytest.mark.usefixtures(self.fixture_name)(test)
 
             # Needed to prevent a failure when settings are applied to the test function
             test.is_hypothesis_test = True  # type: ignore

--- a/test/test_lazy.py
+++ b/test/test_lazy.py
@@ -499,3 +499,31 @@ def test_b(case):
     # And the original one should remain the same
     result = testdir.runpytest()
     result.assert_outcomes(passed=2)
+
+
+def test_parametrized_fixture(testdir, openapi3_base_url):
+    # When the used pytest fixture is parametrized via `params`
+    testdir.make_test(
+        f"""
+schema.base_url = "{openapi3_base_url}"
+
+@pytest.fixture(params=["a", "b"])
+def parametrized_lazy_schema(request):
+    return schema
+
+lazy_schema = schemathesis.from_pytest_fixture("parametrized_lazy_schema")
+
+@lazy_schema.parametrize()
+def test_(case):
+    case.call()
+""",
+    )
+    result = testdir.runpytest("-v")
+    # Then tests should be parametrized as usual
+    result.assert_outcomes(passed=2)
+    result.stdout.re_match_lines(
+        [
+            r"test_parametrized_fixture.py::test_\[a\]\[GET /api/users\]\[P\] PASSED",
+            r"test_parametrized_fixture.py::test_\[b\]\[GET /api/users\]\[P\] PASSED",
+        ]
+    )

